### ISSUE TITLE
Globally exclusive layers

### ISSUE THREAD

**Issue by silvester-pari** [EOX-A/EOxElements#625]
Currently, exclusive layers only check layers on the same "level" (i.e. in the same group) for exclusivity. There might be usecases where we want an exclusive selection for **all** exclusive layers in the entire map, e.g. multiple groups of background layers, and only one of those layers can be shown at a time.

The tricky part here will be to deal with the layer group checkboxes, as when no layer is selected inside a group anymore (because an exclusive layer from another group was set to visible), then the group checkbox doesn't make sense.

Maybe we need a way to either automatically set the checkbox to false if all group layers are hidden, or some way to not show the checkbox at all.

**Comment by StefanBrand** [comment]
This is not the behaviour we are experiencing. We experience globally exclusive layers, when we don't want them.

We want the layers within the group to be exclusive. There should be one layer visible by group. (In this example, e.g. "Mowing Event Conformance" AND "Sentinel-2 Cloudless 2020".

When the app starts, we have one visible layer per group. All of these have `layerControlExclusive: true`. As soon as one is selected, all others become invisible, even in other groups:

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/1ee3723d-4301-40db-a279-d421ba81fc9a" /> 

<img width="250" alt="Image" src="https://github.com/user-attachments/assets/ee42b78f-8953-4a9f-bc39-b7e72cdd9514" />

**Comment by silvester-pari** [comment]
You're right, this is an unwanted effect and a bug, not sure when and how this was introduced 🤔 It seems that this came with the restructuring somewhere. But this gives us the opportunity to now introduce this `globallyExclusiveLayers` functionality.

**Comment by silvester-pari** [comment]
A new property `globallyExclusiveLayers` has been introduced with https://github.com/EOX-A/EOxElements/pull/1753, and the bug mentioned by @StefanBrand has been fixed (by default, layers are exclusive within the group, unless `globallyExclusiveLayers` is set).
